Runtime/Interaction/SnapZonePreviewDrawer.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using UnityEngine;
+
+namespace Innoactive.Creator.XRInteraction
+{
+    [ExecuteInEditMode]
+    public class SnapZonePreviewDrawer : MonoBehaviour
```

Comment:
Missing docs

Reply:
Also, since it is not really meant to be in our public API, maybe all the publics here can be switched to internal.